index.rst

```diff
@@ -87,7 +87,7 @@ visit our `gallery </docs/dev/auto_examples>`__.
       .. image:: _static/sobel-coins-small.png
          :class: coins-sample span6
 
-You can read more in our `user guide </docs/stable/user_guide.html>`__.
+You can read more in our `user guide </docs/stable/user_guide>`__.
```

Comment:
```suggestion
You can read more in our `user guide </docs/stable/user_guide/>`__.
```
Nitpick: maybe add the "/" for consistency? My browser adds it automatically so it's probably the "correct" link.